cmd2/cmd2.py

```diff
@@ -268,21 +269,7 @@ def cmd_wrapper(instance, cmdline):
 
         # Mark this function as having an argparse ArgumentParser (used by do_help)
```

Comment:
Since this isn't just used by do_help anymore, maybe update this comment to say something like "Used to tab complete this command and its subcommands"
This comment appears twice in the file so update both.